### PR TITLE
[php8-compact] Fix Case Activity Message Template to fix test failure…

### PR DESCRIPTION
**Pull request by seamuslee001**
… in php8

Overview
----------------------------------------
This fixes the test failure 

- CRM_Core_BAO_MessageTemplateTest::testCaseActivityCopyTemplate
Undefined array key "contact"

Before
----------------------------------------
Test fails in php8

After
----------------------------------------
Test passes in php8

ping @demeritcowboy @totten @colemanw

### CODE REVIEW

File: xml/templates/message_templates/case_activity_html.tpl [hunk 1]
```diff
@@ -73,27 +73,29 @@
...
+           {if $field.type eq 'Date'}
+            {$field.value|crmDate:$config->dateformatDatetime}
+           {else}
+            {$field.value}
+           {/if}
+          </td>
+         </tr>
+        {/foreach}
+       {/foreach}
+      {/if}
```
Comment (eileenmcnaughton) [inline xml/templates/message_templates/case_activity_html.tpl:98]: This alignment looks a bit wonky - I guess give the state our tpls that's non-blocking

File: xml/templates/message_templates/case_activity_text.tpl [hunk 2]
```diff
@@ -2,16 +2,16 @@
 {ts}Activity Summary{/ts} - {$activityTypeName}
 ===========================================================
 {if !empty($isCaseActivity)}
-{ts}Your Case Role(s){/ts} : {$contact.role}
-{if $manageCaseURL}
+{ts}Your Case Role(s){/ts} : {$contact.role|default:''}
```
Comment (eileenmcnaughton) [inline xml/templates/message_templates/case_activity_text.tpl:5]: I hadn't realised this was an option

Comment (seamuslee001) [reply xml/templates/message_templates/case_activity_text.tpl:5]: yeh @totten raised it here https://chat.civicrm.org/civicrm/pl/jbbd4efzy3yp7ed9y1bjnphnuy

Comment (seamuslee001) [reply xml/templates/message_templates/case_activity_text.tpl:5]: Given this test specifically looks for Your Case Role in the output figured the default was the better way to go

**Pull request merged**